packages/core/src/StrykerCli.ts

```diff
@@ -1,22 +1,44 @@
-import * as program from 'commander';
+import { Command } from 'commander';
 import { getLogger } from 'log4js';
 import { CONFIG_SYNTAX_HELP } from './config/ConfigReader';
 import { initializerFactory } from './initializer';
 import LogConfigurator from './logging/LogConfigurator';
 import Stryker from './Stryker';
+import { DashboardOptions, StrykerOptions, ALL_REPORT_TYPES } from '@stryker-mutator/api/core';
+import { Config } from '@stryker-mutator/api/config';
+import { Logger } from '@stryker-mutator/api/logging';
+
+function deepOption<T extends string, R>(object: { [K in T]?: R }, key: T, valueInterpreter = (value: R) => value) {
```

Comment:
Why is this function not part of your class?

Reply:
Because it's not using `this`. Moreover, I especially don't want to make a mistake and use `this`, since it will be redirected to be something else (the Commander instance or something)